bash_completion

```diff
@@ -1082,15 +1132,19 @@ _mac_addresses()
 #
 _configured_interfaces()
 {
+    local -a files
     if [[ -f /etc/debian_version ]]; then
         # Debian system
+        _comp_expand_glob files '/etc/network/interfaces /etc/network/interfaces.d/*'
         COMPREPLY=($(compgen -W "$(command sed -ne 's|^iface \([^ ]\{1,\}\).*$|\1|p' \
-            /etc/network/interfaces /etc/network/interfaces.d/* 2>/dev/null)" \
+            "${files[@]}" 2>/dev/null)" \
```

Comment:
`${#files[@]}` check (bash 4.2 nounset protection) missing?

Reply:
Right. That's my oversight. Thank you.

Reply:
fixed in 3f86a094a0aa76cec2783f333a1fe2f20933b75e